Sources/Player/Player.swift

```diff
@@ -109,14 +110,20 @@ public extension Player {
 		externalPlayers: [GenericMediaPlayer.Type] = [],
 		credentialsProvider: CredentialsProvider,
 		eventSender: EventSender,
-		userClientIdSupplier: (() -> Int)? = nil
+		userClientIdSupplier: (() -> Int)? = nil,
+		shouldAddLogging: Bool = false
 	) -> Player? {
 		if shared != nil {
 			return nil
 		}
 
 		Time.initialise()
 
+		// When we have logging enabled, we create a logger to be used inside Player module and set it in PlayerWorld.
+		if shouldAddLogging {
+			PlayerWorld.logger = TidalLogger(label: "Player", level: .trace)
+		}
```

Comment:
Also, is there any plan about how are we going to use this? 

- Always? 
- Controlled by a setting? 
- Controlled by a FeatureFlag?
- Enabled only in Beta builds?

Reply:
The consumer of the sdk will decide this.